Cargo.toml

```
@@ -9,7 +9,7 @@ license = "MIT"
 edition = "2018"
 readme = "README.md"
 categories = ["development-tools::ffi", "api-bindings"]
-exclude = ["/robusta-codegen", "/robusta-example", "README.md"]
+exclude = ["/robusta-codegen", "/robusta-example", "README.md", "robusta-android-example"]
```

I know it's a nitpick but... could you add a slash before `robusta-android-example` or remove slashes from the other two folders? I don't matter the choice, it's just for consistency 😅 

```suggestion
exclude = ["/robusta-codegen", "/robusta-example", "README.md", "/robusta-android-example"]
```

```suggestion
exclude = ["robusta-codegen", "robusta-example", "README.md", "robusta-android-example"]
```